docs/settings/task-manager-settings.asciidoc

```diff
@@ -0,0 +1,35 @@
+[role="xpack"]
+[[task-manager-settings-kb]]
+=== Task Manager settings in {kib}
+++++
+<titleabbrev>Task Manager settings</titleabbrev>
+++++
+
+Task Manager runs background tasks by polling for work on an interval.  You can configure its behavior to tune for performance and throughput.
+
+[float]
+[[task-manager-settings]]
+==== Task Manager settings 
+
+[cols="2*<"]
+|===
+| `xpack.task_manager.max_attempts`
+  | The maximum number of times a task will be attempted before being abandoned as failed.  Defaults to 3.
+
+| `xpack.task_manager.poll_interval`
+  | How often, in milliseconds, the task manager will look for more work.  Defaults to 3000 and cannot be lower than 100.
+
+| `xpack.task_manager.max_poll_inactivity_cycles`
+  | How many poll interval cycles can work take before it's timed out.  Defaults to 10.
+
```

Comment:
I'm not sure we should include this.
I don't expect customers to change this - if they go too low it might restart TM prematurely and we've yet to see a deployment where the default wasn't sufficient

Reply:
OK will remove.